src/main/java/frc/robot/commands/DriveTrainCommand.java

```diff
@@ -27,6 +30,10 @@
 	 * @param DriveTrain - The Drive Train Subsystem
 	 */
 	public DriveTrainCommand(DriveTrainSubsystem DriveTrain) {
+		if (Constants.m_RobotState.getCurrentState() != States.ROTATION
+				&& Constants.m_RobotState.getCurrentState() != States.SHOOTING) {
+			Constants.m_RobotState.setState(States.DRIVE);
+		}
```

Comment:
this only does one check.. Needs to be in execute - Also check Position state as well